exchanges/exchanges_test.go

```diff
@@ -118,3 +120,241 @@ func TestSlaveBot(t *testing.T) {
 	// Start server with --exchange-refresh=1m --exchange-expiry=2m
 	testExchanges(true, t)
 }
+
+var initialPoloniexOrderbook = []byte(`[
+        14,
+        8767,
+        [
+                [
+                        "i",
+                        {
+                                "currencyPair": "BTC_BTS",
+                                "orderBook": [
+                                        {
+                                                "0.00011358": "127734.81648491",
+                                                "0.00011359": "667.14834444",
+                                                "0.00011360": "3651.66059723",
+                                                "0.00011361": "200.14590282",
+                                                "0.00011362": "4816.12553510",
+                                                "0.00011363": "37.08390161",
+                                                "0.00011365": "3419.78939376",
+                                                "0.00011366": "8.05270863",
+                                                "0.00011367": "73239.96650974",
+                                                "0.00011368": "7958.06486028",
+                                                "0.00011369": "142.68135365",
+                                                "0.00011370": "24411.40000000",
+                                                "0.00011372": "244147.92356157"
+                                        },
+                                        {
+                                                "0.00001358": "27734.81648491",
+                                                "0.00001359": "67.14834444",
+                                                "0.00001360": "651.66059723",
+                                                "0.00001361": "20.14590282",
+                                                "0.00001362": "816.12553510",
+                                                "0.00001363": "7.08390161",
+                                                "0.00001365": "419.78939376",
+                                                "0.00001366": ".05270863",
+                                                "0.00001367": "3239.96650974",
+                                                "0.00001368": "958.06486028",
+                                                "0.00001369": "42.68135365",
+                                                "0.00001370": "4411.40000000",
+                                                "0.00001371": "44147.92356157"
+                                        }
+                                 ]
+                        }
+                ]
+        ]
+]`)
+
+var poloniexEmptyUpdate = []byte(`[
+    1010
+]`)
+
+var poloniexOrderbookUpdate = []byte(`[
+    14,
+    85537933,
+    [
+        [
+            "o",
+            0,
+            "0.00011358",
+						"0.00000000"
```

Comment:
Peculiar formatting here, but I suppose it doesn't matter.

Reply:
Weird. It's not like that for me. I remember seeing some discrepancy in formatting of the raw strings in database queries a while back. Not sure what causes this.